srp_daemon/srp_daemon.c

```diff
@@ -1991,12 +2045,16 @@ static int ibsrpdm(int argc, char *argv[])
 		}
 	}
 
+	ret = setup_wakeup_fd();
```

Comment:
Please don't add a call to setup_wakeup_fd() here. ibsrpdm doesn't call get_received_signal() and hence doesn't need wakeup_pipe[] to be initialized.